packages/editor/src/components/block-list/style.scss

```diff
@@ -1072,17 +1080,22 @@
 
 .editor-block-list__block .editor-warning {
 	z-index: z-index(".editor-warning");
-	position: absolute;
-	top: -$block-padding - $border-width;
-	right: -$parent-block-padding - $border-width;
-	left: -$parent-block-padding - $border-width;
+	position: relative;
 
-	// Position for nested blocks
+	// Stretch the warning edge to edge.
+	margin-left: -$parent-block-padding - $border-width;
+	margin-right: -$parent-block-padding - $border-width;
+
+	// Stretch the warning less in nested contexts.
 	.editor-block-list__block & {
-		right: -$block-padding - $border-width;
-		left: -$block-padding - $border-width;
+		margin-left: -$block-padding - $border-width;
+		margin-right: -$block-padding - $border-width;
 	}
 
+	// Pull the warning upwards to the edge, and add a negative bottom margin to compensate.
+	margin-bottom: -$block-padding - $border-width;
+	transform: translateY(-$block-padding - $border-width);
```

Comment:
Given this is `position: relative`, I was wondering if we could just use `top:` instead of the more verbose `transform: translateY`.

Reply:
@jasmussen for thoughts

Reply:
I could swear there was a reason involving margins in obscure cases, where translate doesn't affect the margins at all. The margins are essentially kept in place and the content just pulled out.

But in quick testing, it seems like `top: -15px;` accomplishes the same.

On this friday my brain has short circuited. Given I can't reproduce the issue I think I fixed by using translate, I can't find a good argument to keep it. But I have that niggling feeling that it wasn't chosen at random. What do you think?

Reply:
Lets keep it as it is then and get this merged.